src/index.js

```diff
@@ -2,4 +2,21 @@ function sayHello() {
     console.log('Hello');
 }
 
-sayHello();
\ No newline at end of file
+sayHello();
+
+const liveServer = require("live-server");
+
+const params = {
+	port: 8181, // Set the server port. Defaults to 8080.
+	host: "0.0.0.0", // Set the address to bind to. Defaults to 0.0.0.0 or process.env.IP.
+	root: "/public", // Set root directory that's being served. Defaults to cwd.
+	open: false, // When false, it won't load your browser by default.
+	ignore: 'scss,my/templates', // comma-separated string for paths to ignore
+	file: "index.html", // When set, serve this file (server root relative) for every 404 (useful for single-page applications)
```

Comment:
Why not point to the folder containing the index.html  ```suggestion
	file: "/templates/index.html", // When set, serve this file (server root relative) for every 404 (useful for single-page applications)
```